### ISSUE TITLE
Linux sync preview can't be restarted

### ISSUE THREAD

**Issue by kjetil-kilhavn** [jotta/jotta-cli-issues#150]
Make sure you are running the latest version of jotta-cli before reporting an issue.

**jotta-cli release (`jotta-cli version`):**
jottad version    : 0.10.44027
jotta-cli version : 0.10.44027

**Description of problem:**
I installed the unstable version to test the sync folder feature in Linux as it still isn't available in the stable version.
After adding a file to the sync folder it didn't show up in the sync folder (web).
I tried to manually push it using `jotta-cli syc trigger` but got an error message about the sync already running.
So I entered `jotta-cli sync stop` and got an OK message and tried again. I still got the error message about the sync already running. Then I tried `jotta-cli sync start` and also got the error message about the sync already running. 

**Expected:**
It should be possible to stop and start the sync process.

**jotta-cli status (`jotta-cli status`):**
```
------------------------------------------------------------------------------
 Sync      :
------------------------------------------------------------------------------
   Path      : /home/kjetil/Jottacloud
   Files     : 5 (2.76GiB)
   Mode      : manually triggered
   Status    : syncing now...
   Errors    : could not upload: /home/kjetil/Jottacloud/kjetil.kdbx
------------------------------------------------------------------------------
```

**Relevant logs for the issue (`~/.jottad/jottabackup.log or /var/lib/jotta/jottabackup.log ` )**
The upload to the sync folder failed due to missing permission to the file apparently.
```
pid:65461 2021/05/23 11:33:28 not reporting error => download/upload group failed: sync.uploadFile: files.Open(/home/kjetil/Jottacloud/kjetil.kdbx | /home/kjetil/Jottacloud/kjetil.kdbx) : open /home/kjetil/Jottacloud/kjetil.kdbx: permission denied
```

**Traceback**
```

```

**Additional info:**

**Comment by coopersimon** [comment]
Hi, thanks for the report. We'll investigate!

**Comment by coopersimon** [comment]
We just released unstable version 0.10.44290, which fixed some sync status issues. Let us know if it fixed your issue!

**Comment by kjetil-kilhavn** [comment]
Seems to work correctly now. Thank you.
